### PR TITLE
Header/feature/dev

### PR DESCRIPTION
**Pull request by Kaspazza**
Issue #6

### CODE REVIEW

File: src/app/modules/shared/components/header/header.component.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    selector: 'mto-header',
+    templateUrl: './header.component.html',
+    styleUrls: ['./header.component.scss']
+  })
+  export class HeaderComponent {
+    @Input() isLoggedIn = false;
+    @Input() title = 'Example Title';
+    @Input() userPicture = 'assets/img/profile_picture.jpg';
+  }
+
```
Comment (KrzysiekJodlowski) [inline src/app/modules/shared/components/header/header.component.ts]: Whitespace!

File: src/app/modules/shared/shared.module.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  export class SharedModule {
+    static forRoot(): ModuleWithProviders {
+      return {
+        ngModule: SharedModule,
+        providers: [
+        ],
+      };
+    }
+  }
+
```
Comment (KrzysiekJodlowski) [inline src/app/modules/shared/shared.module.ts]: Whitespace!

File: src/app/modules/shared/components/header/header.component.ts (hunk 1 above)
Comment (Kaspazza) [reply src/app/modules/shared/components/header/header.component.ts]: It's tslint rule
https://palantir.github.io/tslint/rules/whitespace/

Here why:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: src/app/modules/shared/shared.module.ts (hunk 2 above)
Comment (Kaspazza) [reply src/app/modules/shared/shared.module.ts]: It's tslint rule
https://palantir.github.io/tslint/rules/whitespace/

Here why:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Pull request merged**